pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
Discussed with @rootfs 

This container will needs to either run as `privileged` OR it can run as UID 0 with and SCC that allows `CAP_SYS_ADMIN`.

@liggitt @smarterclayton - preference?